src/components/Image.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import React from "react";
+import { Image as RNImage } from "react-native";
+import { connect } from "react-redux";
+import type { Connector } from "react-redux";
+import type { ImageDetails } from "../data/image";
+import { getImageDetails as GetImageDetails } from "../data/image";
+import type { FieldRef } from "../data/field-ref";
+
+type OwnProps = {
+  reference: FieldRef
+};
+
+type StateProps = {
+  getImageDetails: string => ?ImageDetails
+};
+
+type Props = OwnProps & StateProps;
+
+export const Image = ({ reference, getImageDetails, ...props }: Props) => (
```

Comment:
I thought might be nice to type the `...props` as the React Native image props. But I couldn't find flow types for the image component. So we would need to create and maintain our own here 😞. I guess it's fine like it is.

Reply:
yeah, I tried looking for that too. Not sure we want to maintain our own copy.

Reply:
Lets see if we miss them when we don't have them here. We can always add them later. 🙂